src/app-layout/__tests__/multi-layout-props.test.tsx

```diff
@@ -0,0 +1,109 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+/* eslint-disable simple-import-sort/imports */
```

Comment:
We only do this in files which use `src/app-layout/__tests__/utils.tsx` because it contains side effects.

All other tests can benefit from the linter rule